app/models/manageiq/providers/embedded_ansible/automation_manager/azure_credential.rb

```diff
@@ -47,7 +47,36 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::AzureCredential <
     :attributes => API_ATTRIBUTES
   }.freeze
 
+  alias_attribute :secret, :auth_key
```

Comment:
Why is this needed?  I'm concerned because it exposes an encrypted column via an alternate attribute, which does things like expose it to reporting, for example.  There are a number of code paths that expect that `Model.encrypted_columns` holds the list.

Reply:
`:secret` is what is passed into from the UI, so we need to support it in some way.  Possibly this isn't required if we just are doing `params_to_attributes` however.  I can look into that.

Reply:
We can also change the UI to not send secret, but instead send auth_key as we expect